### PR TITLE
Wrap ConvBert integration test forward passes with torch.no_grad()

### PR DESCRIPTION
**Pull request by daspartho**
# What does this PR do?
This PR wraps forward passes in ConvBert integration tests with `torch.no_grad()`, as proposed in issue #14642. This avoids the computation of unnecessary gradients during inference.

## Before submitting
- [ ] This PR fixes a typo or improves the docs.
- [X] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests), Pull Request section?
- [X] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)?
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

## Who can review?
@LysandreJik could you please check it?
Thanks :)

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
Thank you!
